DOC: clarify installation (#259)
jonescompneurolab/hnn-core: README.rst

Installation
============

> We recommend the `Anaconda Python distribution <https://www.anaconda.com/products/individual>`_.

Reviewer: Should we mention a recommended Python minor version? I guess that largely depends on what NEURON is built against.

Author: I think Neuron doesn't work with 3.9 as far as I know ... I hope Anaconda doesn't distribute bleeding edge Python

Reviewer: It might... We could recommend creating an env with python<=3.8? Gets a bit messy though.

Author: yeah let's leave it like this for now and see how many users stumble upon that issue

To install ``hnn-core``, you first need to install its dependencies::

	$ conda install numpy matplotlib scipy

Additionally, you would need Neuron which is available here:
`https://neuron.yale.edu/neuron/ <https://neuron.yale.edu/neuron/>`_.

> We recommend that you install it via pip::
> 
> 	$ pip install NEURON
> 
> Note that if you choose to install Neuron using the traditional installer, it may be hard
> to manage it with virtual environments in Anaconda. For this reason, we recommend the
> pip installer.
> 

Reviewer: ```suggestion
Note installing Neuron using the traditional installer may make it hard to manage
within virtual environments such as Anaconda. For this reason, we recommend the
pip installer:

	$ pip install NEURON

```
